testsuite/integration/manualmode-expansion/src/test/java/org/wildfly/test/manual/mvc/MVCKrazoSmokeTestCase.java

```diff
@@ -0,0 +1,171 @@
+import org.jboss.as.test.integration.common.HttpRequest;
+import org.jboss.as.test.shared.ServerReload;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.dmr.ModelNode;
+import org.jboss.logging.Logger;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.descriptor.api.Descriptors;
+import org.jboss.shrinkwrap.descriptor.api.webapp25.WebAppDescriptor;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+//@ServerSetup(MVCKrazoSmokeTestCase.Setup.class) TODO migrate this to testsuite/integration/basic and let WFARQ call the ServerSetupTask
+public class MVCKrazoSmokeTestCase {
+
+    private static final Logger log = Logger.getLogger(MVCKrazoSmokeTestCase.class.getName());
+
+    // TODO migrate this to testsuite/integration/basic and drop the ContainerController aspect used in manualmode
+    private static final String CONTAINER = "stability-preview";
+    // TODO migrate this to testsuite/integration/basic and let WFARQ call the ServerSetupTask via @ServerSetup
+    private static final ServerSetupTask SETUP_TASK = new MVCKrazoSmokeTestCase.Setup();
+
+    private static final String DEPLOYMENT_NAME = "MVCKrazoSmokeTestCase";
+
+    private static final String MVC_EXTENSION = "org.wildfly.extension.mvc-krazo";
+    private static final String MVC_SUBSYSTEM = "mvc-krazo";
+
+    @Deployment(name = DEPLOYMENT_NAME, testable = false, managed = false) // TODO migrate this to testsuite/integration/basic and change to managed = true
+    @TargetsContainer(CONTAINER)
+    public static WebArchive createDeployment() {
+        Descriptors.create(WebAppDescriptor.class);
+        return ShrinkWrap.create(WebArchive.class, DEPLOYMENT_NAME + ".war")
+                .addClass(MVCKrazoApplication.class)
+                .addClass(MVCKrazoSmokeController.class)
+                .addClass(CdiBean.class)
+                .addAsWebInfResource(new StringAsset("<html>" +
+                                "<p>First Name = [${cdiBean.firstName}]</p>\n" +
+                                "<p>Last Name = [${lastName}]</p>\n" +
+                                "</html>"),
+                        "views/view.jsp");
+    }
+    // TODO migrate this to testsuite/integration/basic and drop the ContainerController aspect used in manualmode
+    @ArquillianResource
+    private static ContainerController containerController;
+
+    // TODO migrate this to testsuite/integration/basic and drop the manual deployment aspect used in manualmode
+    @ArquillianResource
+    private static Deployer deployer;
+
+    // TODO migrate this to testsuite/integration/basic and inject the http listener URL
+//    @ArquillianResource
+//    private URL url;
+
+
+    // TODO migrate this to testsuite/integration/basic and drop this
+    @Test
+    @InSequence(-1)
```

Comment:
This should not block merging, but any reason not to do this in a `@Before` method?